numpy/core/src/multiarray/datetime.c

```diff
@@ -2035,7 +2033,7 @@ metastr_to_unicode(PyArray_DatetimeMetaData *meta, int skip_brackets)
 /*
  * Adjusts a datetimestruct based on a seconds offset. Assumes
  * the current values are valid.
- */
+
 NPY_NO_EXPORT void
 add_seconds_to_datetimestruct(npy_datetimestruct *dts, int seconds)
```

Comment:
disabled this function entirely in last commit to confirm it is unused.  any objection to ripping it out?  OK to consider this out of scope.

Reply:
FYI another way that might help check this would be to declare the function as static and use gcc's -Wunused-function flag
That flag might be a part of -Wall or -Wextra so you might not need an extra flag at all, but it only warns for static functions

Reply:
Ah, sorry for not noticing this.  I am happy to delete it here or otherwise.  The issue is that we have `NPY_NO_EXPORT`  in a few places too much.  It should be `static` and then our CI would already take note.